environment/scripts/setup_data.py

```diff
@@ -35,6 +87,24 @@ def download_gtex_v8_data(**kwargs):
     )
 
 
+def download_multiplier_recount2_data(**kwargs):
+    """
+    This method downloads the recount2 data used in MultiPLIER.
+    """
+    get_file_from_zip(
+        zip_file_url="https://ndownloader.figshare.com/files/10881866",
+        zip_file_path=Path(
+            conf.RECOUNT2["DATA_RDS_FILE"].parent, "recount2_PLIER_data.zip"
+        ).resolve(),
+        zip_file_md5="f084992c5d91817820a2782c9441b9f6",
```

Comment:
Is this MD5 hash a static hash for recount2 data?

Reply:
Yes, it's the MD5 hash taken from figshare.